src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/StaffCategoryCaseTest.java

```diff
@@ -494,6 +496,167 @@ void shouldDeleteApprovedRequestedRoleForCase_V1_1() {
         });
     }
 
+    @Test
+    void shouldApproveCaseRequestedRoles_CCD_1_0_NotFromCCD() {
+        RoleAssignment requestedRole1 = getRequestedCaseRole(RoleCategory.PROFESSIONAL, "[PETSOLICITOR]",
+                                                             SPECIFIC, "caseId",
+                                                             "1234567890123456", CREATE_REQUESTED);
+        assignmentRequest.setRequestedRoles(List.of(requestedRole1));
+        FeatureFlag featureFlag  =  FeatureFlag.builder().flagName(FeatureFlagEnum.CCD_1_0.getValue())
+            .status(true).build();
+        featureFlags.add(featureFlag);
+
+        executeDroolRules(List.of(buildExistingRoleForIAC(requestedRole1.getActorId(),
+                                                          "senior-tribunal-caseworker"),
+                                  buildExistingRoleForIAC(assignmentRequest.getRequest().getAssignerId(),
+                                                          "tribunal-caseworker"),
+                                  buildExistingRoleForIAC(assignmentRequest.getRequest().getAssignerId(),
+                                                          "[PETSOLICITOR]")));
+        //assertion
+        assignmentRequest.getRequestedRoles().forEach(ra -> assertEquals(Status.REJECTED, ra.getStatus()));
+    }
+
+    @Test
+    void shouldApproveCreaterCaseRequestedRoles_CCD_1_0_FromCCD() {
+        verifyCreateCaseRequestedRole_CCD_1_0("[CREATOR]", "ccd_data", RoleCategory.JUDICIAL);
+        verifyCreateCaseRequestedRole_CCD_1_0("[CREATOR]", "aac_manage_case_assignment",
+                                              RoleCategory.LEGAL_OPERATIONS);
+    }
+
+    @Test
+    void shouldApproveCreateCaseRequestedRoles_CCD_1_0_FromCCD() {
```

Comment:
Junit test name is misleading. It should be like shouldApprovePetSolicitiorCaseRole_FromCCD()